src/Jackett.Common/Content/index.html

```diff
@@ -127,6 +127,10 @@ <h3>Jackett Configuration</h3>
             <span class="input-header">Base path override: </span>
             <input id="jackett-basepathoverride" class="form-control input-right" type="text" value="" placeholder="/jackett">
         </div>
+        <div class="input-area">
+            <span class="input-header">Base URL override: </span>
+            <input id="jackett-baseurloverride" class="form-control input-right" type="text" value="" placeholder="jackett:9117">
```

Comment:
```suggestion
            <input id="jackett-baseurloverride" class="form-control input-right" type="text" value="" placeholder="http://jackett:9117">
```

Reply:
Please change the stamp on line 750 of the index.html so that browsers reload the custom.js instead of using the cache. Thanks.

Reply:
> Please change the stamp on line 750 of the index.html so that browsers reload the custom.js instead of using the cache. Thanks.

Hi @garfield69, I addressed this with https://github.com/Jackett/Jackett/pull/12882/commits/32c5c3a0dd49331d776ddc226bc1e6bfda2ba9a7 - hope it's fine!